src/Sylius/Bundle/ResourceBundle/DependencyInjection/AbstractResourceExtension.php

```diff
@@ -261,6 +270,32 @@ protected function loadDatabaseDriver(array $config, LoaderInterface $loader, Co
     }
 
     /**
+     * @param array $config
+     * @param ContainerBuilder $container
+     */
+    protected function configureTranslations(array $config, ContainerBuilder $container)
```

Comment:
Yeah, I wonder if you could do like how Symfony does with PrependExtensionInterface. Have TranslatableExtensionInterface that is in the TranslationBundle and search all the bundles in a compiler pass also in TranslationBundle. Then you can even provide the binary for each configure as well:

```php
interface TranslatableExtensionInterface
{
    const CONFIGURE_TRANSLATION = 32;
}
```